spec/app_support/notification_sender_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe NotificationSender, :aggregate_failures do
+  let(:user) { create(:user) }
+
+  let(:facility) { create(:facility) }
+
+  let(:facility_account) do
+    facility.facility_accounts.create(attributes_for(:facility_account))
+  end
+
+  let(:item) do
+    facility
+    .items
+    .create(attributes_for(:item, facility_account_id: facility_account.id))
+  end
+
+  let(:account) { create(:nufs_account, :account_users_attributes => account_users_attributes_hash(:user => user), :facility_id => facility.id) }
```

Comment:
Hashrockets. Also, considering how often we use this pattern, I feel like we could replace the `account_users_attributes:` with transient attributes (the `:setup_account` factory seems to be almost there).